### ISSUE TITLE
Add the option for keyboard navigation with Virtualizer

### ISSUE THREAD

**Issue by michael-lauridsen** [inokawa/virtua#579]
**Is your feature request related to a problem? Please describe.**
I have the issue that keyboard navigation does not work properly with the Virtualizer component for items not being rendered. This causes an issue with accessibility when trying to enable keyboard navigation in a long list. 
The issue can be reproduced in the official demo: [Radix Select with Virtualizer](https://inokawa.github.io/virtua/?path=/story/advanced-with-radix-ui--select) - For example if you open the select and press B you will only cycle through the shown items, but never the ones further down in the list. 

**Describe the solution you'd like**
Option to reactively set the index.

**Describe alternatives you've considered**
I have tried to handle finding the index of the child item that starts with the key pressed, but I ended being stuck as I could not find a way afterwards to control the list.

**Additional context**
![bfe15326-2798-41c9-9418-0ef9bda0e3bc](https://github.com/user-attachments/assets/c134d08f-104d-41fa-898c-4fab79fd740c)

**Comment by inokawa** [comment]
> Option to reactively set the index or a scrollTo function.

Could you explain it in detail?
